### ISSUE TITLE
Include in clj-refactor itself?

### ISSUE THREAD

**Issue by expez** [maio/discover-clj-refactor.el#1]
Hey

This looks pretty great!  We discussed adding something like this to aid with discovery in clj-refactor a while back, but nothing came of it.  Would you be interested in submitting a pr with this functionality to cljr itself?

This popup only appears when you hit the associated keybinding, in this case `C-c j`, right?

@magnars, @benedekfazekas: Do you remember why we didn't just go with `discover.el` but waited for a lib much like `discover.el` to be extracted from magit?

**Comment by magnars** [comment]
No, I think it was a preference of the person coming to use with the suggestion - I can't remember the specifics.

**Comment by maio** [comment]
Sure, I have no problem with including it directly in clj-refactor. It makes a lot of sense to me. I'll create PR.

There is one (pretty big) issue with `discover-clj-refactor.el` right now. Refactorings which use multiple cursors don't work. See https://github.com/magnars/multiple-cursors.el/issues/147

Right now I'm using quick/dirty fix for this locally. I changed `(mc/maybe-multiple-cursors-mode)` call in `cljr-introduce-let` to `(run-with-timer 0 nil 'mc/maybe-multiple-cursors-mode)`.

Do you guys think this would be acceptable change to make it work? (cc: @magnars)

**Comment by magnars** [comment]
Sounds like a acceptable workaround to me. Maybe extract it into `cljr--maybe-multiple-cursors` with some explanatory prose as docstring?

**Comment by maio** [comment]
OK, will do.
